client/webserver/http.go

```diff
@@ -96,18 +95,14 @@ func (s *WebServer) handleRegister(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// TODO: rework register page for paying with other assets.
-	feeAssetID, _ := dex.BipSymbolID("dcr")
-	feeWalletStatus := s.core.WalletState(feeAssetID)
-	feeWalletExists := feeWalletStatus != nil
-	feeWalletOpen := feeWalletExists && feeWalletStatus.Open
+	// feeAssetID, _ := dex.BipSymbolID("dcr")
+	// feeWalletStatus := s.core.WalletState(feeAssetID)
+	// feeWalletExists := feeWalletStatus != nil
+	// feeWalletOpen := feeWalletExists && feeWalletStatus.Open
 
 	switch {
 	case !cArgs.UserInfo.Initialized:
 		data.InitStep = true
-	case !feeWalletExists:
-		data.WalletStep = true
-	case !feeWalletOpen:
-		data.OpenStep = true
 	default:
 		data.DEXStep = true
```

Comment:
We can simplify things a lot now that wallet setup and unlock are not an automated part of the workflow.  Now: (a) wallet *setup* is being initiated by user interaction on the fee assets table, and (b) wallet *unlock* happens automatically (in `(*Core).Register`) when the user submits the registration form with their app pass.

Thus, consider:

```go

type registerTmplData struct {
	CommonArguments
	Initialized bool
}

func (s *WebServer) handleRegister(w http.ResponseWriter, r *http.Request) {
...

	data := &registerTmplData{
		CommonArguments: *cArgs,
		Initialized:     cArgs.UserInfo.Initialized,
	}

	s.sendTemplate(w, "register", data)
}
```

Then modify register.tmpl: Only the `Initialized` field is needed now (if initialized, it's implied that it's on the DEX entry step because wallet interactions come after that as I've indicated at the top of this comment).  So opposite semantics to the previous `InitStep` field and no need for any of the other booleans afaict.